media/js/firefox/whatsnew/whatsnew-110-features.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+(function () {
+    'use strict';
+
+    var client = Mozilla.Client;
+
+    function handleOpenProtectionReport(e) {
+        e.preventDefault();
+
+        window.dataLayer.push({
+            event: 'in-page-interaction',
+            eAction: 'link click',
+            eLabel: 'See what Firefox has blocked for you'
+        });
+
+        Mozilla.UITour.showProtectionReport();
+    }
+
+    if (client.isFirefoxDesktop) {
+        if (client._getFirefoxMajorVersion() >= 70) {
```

Comment:
Question (non-blocking): is this `if/else` condition redundant, seeing as this page will be shown only to FF 110?

Reply:
You're right, this was a carryover from a different page. We don't need this on WNP (the link falls back to a SUMO link in non-Firefox, we don't show any different content for old versions of Fx).